doc/sphinx/source/recipes/recipe_arctic_ocean.rst

```diff
@@ -0,0 +1,268 @@
+.. _recipes_arctic_ocean:
+
+Recipe for evaluating Arctic Ocean
```

Comment:
Could you maybe call this differently to avoid having an index page in the documentation where all recipes start with "Recipe for.. ". e.g. "Arctic Ocean Evaluation"

Reply:
Changed to "Arctic Ocean Evaluation".